main/res/values/strings.xml

```diff
@@ -857,6 +857,8 @@
     <string name="cache_log_images_title">Log images</string>
     <string name="cache_log_image_default_title">Photo</string>
     <string name="cache_personal_note">Personal note</string>
+    <string name="cache_menu_preventWaypointsFromNote">Prevent waypoints from note</string>
+    <string name="cache_menu_allowWaypointExtraction">Allow waypoints from note</string>
```

Comment:
Where is the extra parameter "cache_menu_allowWaypointExtraction" used?

Reply:
Those two lines are for the cache menu - enabling / disabling waypoint extraction from personal note.

Reply:
but it is a checkbutton, isn't it?

Reply:
No, the strings starting with "cache_menu_" refer to the new menu entry at the end of the cache menu, whereas the new string "cache_personal_note_preventWaypointsExtraction" is for the checkbox in the personal note edit dialog.

Reply:
Understand, ok.